irondb/administration/compacting-numeric-rollups.md

```diff
@@ -4,10 +4,11 @@ Although numeric shards can be configured with [retention windows](../getting-st
 
 Compaction is performed by running the `shard_compactor` tool. It has two required arguments:
 
-* `-d <nntbs_dir>` - The path where NNTBS shards are stored. This is typically found under `/irondb/nntbs`, or `/snowth/nntbs` on Apica Inside deployments. The directory name matches the node's cluster UUID.
```

Comment:
We should leave the /snowth pathing here.   Remember EA may use this same docs.

Reply:
Restored, though with slightly different wording since the Inside product is no longer available.